python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -248,6 +371,157 @@ def getObservationMetaData(self, obsHistID=None, expDate=None, fieldRA=None, fie
                                                                     for column in self.active_columns if column[2] is not None]))
                                           for pointing in results]
 
-
         return obs_output
 
+    @staticmethod
+    def ObservationMetaDataForPointing(OpSimPointingRecord, columnMap,
+                                       OpSimColumns=None,
+                                       boundLength=1.75, boundType='circle'):
+        """
+        Return instance of ObservationMetaData for an OpSim Pointing record
+        from OpSim.
+
+        Parameters
+        ----------
+        OpSimPointingRecord : Dictionary, mandatory
```

Comment:
I see.  This only works on one OpSim record at a time.  There should also be a method to take a recarray of OpSim records and return a list of ObservationMetaDatas.